docs/modding/suitmodding/addinglocals copy 2.mdx

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
I'm gonna assume these copy files are just drafts written by ChatGPT and not intended to be on the website?

Reply:
The copy files are me making some changes to when new information gets introduced. Like in the blobs folder it's all over the place so as not to overwhelm the user with technical jargon and kind of reel them in the process while they're learning stuff